### PR TITLE
update devcloud env setup file for INC TensorFlow sample

### PR DESCRIPTION
**Pull request by nazneenn**
## Type of Change

update env setup script

## Description

added pip install for devcloud setup

## Expected Behavior & Potential Risk

 Conda+PIP installation is recommended

## How has this PR been tested?

tested on devcloud

## Dependency Change?

added pip install INC and TF

### CODE REVIEW

**Review by NeoZhangJianyu — APPROVED** [review]
Approve

File: examples/notebook/tensorflow/alexnet_mnist/inc_quantize_model.py [hunk 1]
```diff
@@ -48,22 +48,30 @@ def compare_ver(src, dst):
...
         conf=config,
         calib_dataloader=dataloader,
         eval_dataloader=dataloader)
+    
 
     return q_model
 
 
-config = PostTrainingQuantConfig()
+#yaml_file = "alexnet.yaml"
```
Comment (NeoZhangJianyu) [inline examples/notebook/tensorflow/alexnet_mnist/inc_quantize_model.py:71]: rm this line, useless.

File: examples/notebook/tensorflow/alexnet_mnist/run_in_intel_devcloud.sh [hunk 2]
```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 echo "Enable Conda Env."
-source ${ONEAPI_ROOT}/intelpython/python3.9/etc/profile.d/conda.sh
+source /opt/intel/oneapi/setvars.sh
```
Comment (NeoZhangJianyu) [inline examples/notebook/tensorflow/alexnet_mnist/run_in_intel_devcloud.sh:4]: In devcloud, this path is not working.
Please use "${ONEAPI_ROOT}/intelpython/python3.9/etc/profile.d/conda.sh" if verify is passed.

File: examples/notebook/tensorflow/alexnet_mnist/inc_quantize_model.py (hunk 1 above)
Comment (nazneenn) [reply examples/notebook/tensorflow/alexnet_mnist/inc_quantize_model.py:71]: Thank you for letting me know.. i will do the necessary changes.

File: examples/notebook/tensorflow/alexnet_mnist/run_in_intel_devcloud.sh (hunk 2 above)
Comment (nazneenn) [reply examples/notebook/tensorflow/alexnet_mnist/run_in_intel_devcloud.sh]: Hi,
After verifying, I implemented the changes mentioned above.

Comment (NeoZhangJianyu) [reply examples/notebook/tensorflow/alexnet_mnist/run_in_intel_devcloud.sh]: OK! please ignore my comments.

File: examples/notebook/tensorflow/alexnet_mnist/inc_quantize_model.py (hunk 1 above)
Comment (nazneenn) [reply examples/notebook/tensorflow/alexnet_mnist/inc_quantize_model.py]: Done all the necessary changes.

**Pull request merged**